q2_moshpit/assets/busco/index.html

```diff
@@ -1,138 +1,212 @@
-                href="all_batch_summeries.csv"
-                >BUSCO batch summary for all samples (csv)</a
-              >
-              <a class="btn btn-outline-secondary" href="BUSCO_plots.zip"
-                >BUSCO plots for all samples (zip)</a
-              >
+<script src="https://cdn.jsdelivr.net/npm//vega@5" type="text/javascript"></script>
+<script src="https://cdn.jsdelivr.net/npm//vega-lite@5.16.3" type="text/javascript"></script>
+<script src="https://cdn.jsdelivr.net/npm//vega-embed@6" type="text/javascript"></script>
+<link crossorigin="anonymous" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" integrity="sha256-YvdLHPgkqJ8DVUxjjnGVlMMJtNimJ6dYkowFFvp4kKs=" rel="stylesheet"/>
+{% endblock %}
+
+{% block tabcontent %}
+<script crossorigin="anonymous" integrity="sha256-9SEPo+fwJFpMUet/KACSwO+Z/dKMReF9q4zFhU/fT9M=" src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
+
+<br>
+<div class="row" style="justify-content: space-evenly; align-items: start;">
+    <div class="row" style="margin-bottom: 10px">
+        <div class="col-lg-12" style="margin: unset">
+            <div class="card">
+                <h5 class="card-header">BUSCO summary statistics</h5>
+                <div class="card-body">
+                    <p>
+                        The table below shows the summary statistics of the BUSCO marker count
+                        fractions for the categories <b>Single</b>, <b>Duplicated</b>, <b>Fragmented</b>,
+                        <b>Missing</b>, and <b>Complete</b>. The statistics are calculated for
+                        the categories across <b>all MAGs</b> in <b>all samples</b>.
+                    </p>
+                    <table id="summaryTable" class="table table-hover">
+                    <thead>
+                        <tr>
+                            <th scope="col">#</th>
+                            <th scope="col">% single</th>
+                            <th scope="col">% duplicated</th>
+                            <th scope="col">% fragmented</th>
+                            <th scope="col">% missing</th>
+                            <th scope="col">% complete</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr id="rowMinimum">
+                          <th scope="row">Minimum</th>
+                        </tr>
+                        <tr id="rowMedian">
+                          <th scope="row">Median</th>
+                        </tr>
+                        <tr id="rowMean">
+                          <th scope="row">Mean</th>
+                        </tr>
+                        <tr id="rowMaximum">
+                          <th scope="row">Maximum</th>
+                        </tr>
+                        <tr id="rowTotal">
+                          <th scope="row">Total count</th>
```

Comment:
```suggestion
                          <th scope="row">Number of MAGs</th>
```
I might be overkill but maybe this is more informative. I was a little puzzled when i saw "Total count"